docs/misc/iaq.md

```diff
@@ -1,10 +1,63 @@
 Infrequently Asked Questions
 ============================
 
-### For troubleshooting, how can I disable Xen Meltdown mitigations?
+## Troubleshooting
+
+### How can I disable Xen Meltdown mitigations?
 
 Set `xpti=false` option in Xen command line (xen.gz option in grub, or options= line in xen.cfg for UEFI).
 
+### How can I switch R4.0 stubdomains back to qemu-traditional?
+
+~~~
+qvm-features VMNAME linux-stubdom ''
+~~~
+
+### How can I upgrade everything to testing?
+
+dom0: `sudo qubes-dom0-update --enablerepo=qubes-dom0-current-testing --clean` (or --check-only instead for dom0).
+
+fedora: `sudo dnf update --enablerepo=qubes-vm-*-current-testing --refresh`
+
+debian/whonix: `sudo apt-get update -t *-testing && sudo apt-get dist-upgrade -t *-testing`
+
+This way, you don't need to edit any files for debian/whonix to get the testing.
+If you also want to increase reliability further, you can make a dependency/cache check with "sudo apt-get check", which is normally very quick.
+For that, under debian/whonix do: `sudo apt-get check && sudo apt-get update -t *-testing && sudo apt-get dist-upgrade -t *-testing`.
+
+### VM fail to start after hard power off
+
+I realized that some VMs refuse to start after a hard power-off (hold power button for 10s).
+When running `qvm-start test` I get `vm-test-private missing`.
+But this thin volume is actually there.
+Also the volume `vm-test-private-snap` is still present.
+
+Try this in dom0:
+~~~
+sudo pvscan --cache --activate ay
+sudo systemctl restart qubesd
+qvm-start test
+~~~
+
+### Slow VM startup
+
+Use tools like 'systemd-analyze blame' as your guide.
+
+Another service that shows up with significant time is wpa_supplicant. 
+You can have it start only for network VMs by creating `/lib/systemd/system/wpa_supplicant.service.d/20_netvms` with the following:
+~~~
+[Unit]
+ConditionPathExists=/var/run/qubes/this-is-netvm
+~~~
+
+### Xen passthrough compatible video cards
+
+- https://en.wikipedia.org/wiki/List_of_IOMMU-supporting_hardware#AMD
+- http://www.overclock.net/t/1307834/xen-vga-passthrough-compatible-graphics-adapters
+- https://wiki.xenproject.org/wiki/Xen_VGA_Passthrough_Tested_Adapters#ATI.2FAMD_display_adapters
+
```

Comment:
Does this belong to "troubleshooting" ?
BTW that's the section I wanted to link to the hardware selection doc.

Reply:
Which section should I go in? (And I know, but please do a PR against that doc.)

Reply:
Not arguing, just not sure where else to put it. Maybe there should be another section, but I don't know what to call it! Merging for now.

Reply:
don't worry ; I was also wondering where to put it - thought about a 'misc' section - but in the end 'troubleshooting' seemed OK.